### PR TITLE
Comments refactor and typo fixes

### PR DESCRIPTION
**Pull request by QilongTang**
### Purpose

For a cleaner code base and better API documentation.
MAGN-9914 API Documentation Priority 3 - Part Deux
http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-9914

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.

### Reviewers

@ramramps 

### FYIs
@sm6srw @mjkkirschner @jnealb 
Will cherry pick this to 1.0 later.

### CODE REVIEW

File: src/DynamoCore/Configuration/ExecutionSession.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ public ExecutionSession(Scheduler.UpdateGraphAsyncTask updateTask, DynamoModel m
         public string CurrentWorkspacePath { get; private set; }
 
         /// <summary>
-        /// Gets session parameter value for the given parameter name.
+        /// Queries session parameter value for the given parameter name.
```
Comment (ramramps) [inline src/DynamoCore/Configuration/ExecutionSession.cs]: Queries = Returns. Please change it in all the place.

File: src/DynamoCore/Core/CustomNodeManager.cs [hunk 2]
```diff
@@ -445,7 +445,7 @@ public bool TryGetFunctionDefinition(Guid id, bool isTestMode, out CustomNodeDef
         }
         
         /// <summary>
-        ///     Tells whether the custom node's unique identifier is inside of the manager (initialized or not)
+        ///     Indicates whether the custom node's unique identifier is inside of the manager (initialized or not)
```
Comment (ramramps) [inline src/DynamoCore/Core/CustomNodeManager.cs]: Returns true if the custom nodes unique identifier is inside the manager.

File: src/DynamoCore/Core/CustomNodeManager.cs [hunk 3]
```diff
@@ -454,7 +454,7 @@ public bool Contains(Guid guid)
         }
 
         /// <summary>
-        ///     Tells whether the custom node's name is inside of the manager (initialized or not)
+        ///     Indicates whether the custom node's name is inside of the manager (initialized or not)
```
Comment (ramramps) [inline src/DynamoCore/Core/CustomNodeManager.cs]: Indicates = Returns true if .....

File: src/DynamoCore/Engine/EngineController.cs [hunk 4]
```diff
@@ -303,8 +303,7 @@ internal List<Guid> PreviewGraphSyncData(IEnumerable<NodeModel> updatedNodes, bo
         }
 
         /// <summary>
-        /// Return true if there are graph sync data in the queue waiting for
-        /// being executed.
+        /// Return true if there are graph sync data in the queue waiting to be executed.
```
Comment (ramramps) [inline src/DynamoCore/Engine/EngineController.cs]: Returns

File: src/DynamoCore/Library/FunctionDescriptor.cs [hunk 5]
```diff
@@ -136,12 +136,12 @@ public FunctionDescriptor(FunctionDescriptorParams funcDescParams)
         public bool IsOverloaded { get; set; }
 
         /// <summary>
-        ///     Full path to the assembly the defined this function
+        ///     Full path to the assembly which defined this function
```
Comment (ramramps) [inline src/DynamoCore/Library/FunctionDescriptor.cs]: Returns full path of the assembly

File: src/DynamoCore/Models/RunSettings.cs [hunk 6]
```diff
@@ -42,7 +42,7 @@ public int RunPeriod
         }
 
         /// <summary>
-        /// The current RunType.
+        /// The current Run Type. E.g. Manual, automatic.
```
Comment (ramramps) [inline src/DynamoCore/Models/RunSettings.cs]: Returns....

File: test/Engine/FFITarget/ManagedElementExample.cs [hunk 7]
```diff
@@ -155,7 +155,7 @@ public bool IsRevitDeleted(T id)
 
 
         /// <summary>
-        /// This method tells the life cycle 
+        /// This method indicates the life cycle
```
Comment (ramramps) [inline test/Engine/FFITarget/ManagedElementExample.cs]: which life cycle?

**Comment by QilongTang** [comment]
@ramramps I believe I unified all the use here.

**Comment by ramramps** [comment]
LGTM/

**Pull request merged**